### DISCUSSION TITLE
Question About Hiding/Showing a Sidebar Using Tailwind and Preline

### DISCUSSION THREAD

**Discussion by sorrowmiso** [htmlstreamofficial/preline#451] [Q&A]
I'm new to using Tailwind and Preline. I've been trying various things, but on the sample page https://preline.co/pro/analytics/index.html, I want to implement a feature where the 'Quick Actions' section can be hidden or shown. I've tried using offcanvas or dropdown functionalities for a few days, but they don't seem to work correctly.

Is there any example page or reference I could use for this task?

**Comment by larhou** [comment]
I have the same question. 

On the Preline homepage various sidebar toggle buttons are used. See attachments. 

Do the Preline documentation provide any examples on this can be enabled (javascript + css)?

<img width="974" alt="Sidebar-closed" src="https://github.com/user-attachments/assets/5474e504-dd3d-434b-a53a-a34378088056">

<img width="972" alt="Sidebar-open" src="https://github.com/user-attachments/assets/0f528c36-da59-4a75-99a7-39ecca4c5c16">
